Zend/tests/traits/gh15753.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+GH-15753: Overriding readonly properties from traits don't allow assignment from the child
```

Comment:
The test works fine on **unpatched** master, but fails on PHP-8.2. Is this a backport?

Reply:
No, this works on `master` because of the changes to `readonly` in https://wiki.php.net/rfc/asymmetric-visibility-v2#relationship_with_readonly. Namely, `readonly` properties can, by default, now be initialized from all sub-classes. For `master`, we should add a better test.